paymentsheet/src/androidTest/java/com/stripe/android/paymentsheet/FlowControllerTest.kt

```diff
@@ -187,7 +187,9 @@ internal class FlowControllerTest {
             page.fillOutCardDetails()
 
             page.clickPrimaryButton()
-            assertThat(testContext.configureCallbackTurbine.awaitItem()?.label).endsWith("4242")
+            val paymentOption = testContext.configureCallbackTurbine.awaitItem()
+            assertThat(paymentOption?.label).endsWith("4242")
+            assertThat(paymentOption?.paymentMethodType).endsWith("card")
```

Comment:
paymentOption paymentMethodType should be asserted using isEqualTo. We don't expect anything to be prepended unlike card labels.

```suggestion
assertThat(paymentOption?.paymentMethodType).isEqualTo("card")
```